prediction/src/tests/test_endpoints.py

```diff
@@ -4,13 +4,13 @@
 
     Provides unit tests for the API endpoints.
 """
+from flask import url_for
 from functools import partial
 import json
-
 import pytest
-
-from flask import url_for
-from src.factory import create_app
+import sys
+sys.path.append('.')
+from src.factory import create_app  # NOQA
```

Comment:
What about just tweaking the `Makefile` to add a step `touch [whatever]/__init__.py` then making the docs then removing them?